docs/textbook/Central_Limit_Theorem3.ipynb

```diff
@@ -0,0 +1,150 @@
+   "source": [
+    "# Central Limit Theorem\n",
+    "This is an example using the `nbinteract.hist` method to create an interactive histogram. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# HIDDEN\n",
+    "import nbinteract\n",
+    "import numpy as np\n",
+    "\n",
+    "from scipy import stats\n",
+    "from datascience import *"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "    <thead>\n",
+       "        <tr>\n",
+       "            <th>Coin</th>\n",
+       "        </tr>\n",
+       "    </thead>\n",
+       "    <tbody>\n",
+       "        <tr>\n",
+       "            <td>0   </td>\n",
+       "        </tr>\n",
+       "    </tbody>\n",
+       "        <tr>\n",
+       "            <td>1   </td>\n",
+       "        </tr>\n",
+       "    </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "Coin\n",
+       "0\n",
+       "1"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "coin = Table().with_column('Coin', np.arange(0,2))\n",
+    "coin"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`coin_clt_hist` is a response function that simulates the bootstrap process by sampling the mean of multiple series of coin flips. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def coin_clt_hist(num_flips, repetitions=100):\n",
+    "    net_gain_coin_flips = make_array()\n",
+    "    for i in np.arange(repetitions):\n",
+    "        new_coin= coin.sample(num_flips)\n",
+    "        new_net_gain_coin_flip = new_coin.column('Coin').mean()\n",
+    "        net_gain_coin_flips = np.append(net_gain_coin_flips, new_net_gain_coin_flip)\n",
+    "    return net_gain_coin_flips"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "6a2a3b31faf64fbcb0d22c0ba5579db6",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "A Jupyter Widget"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "afa6c3e86f814e3d8cf721b485974bda",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "A Jupyter Widget"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "nbinteract.hist(coin_clt_hist, num_flips=(0,500), repetitions = (0,500))"
```

Comment:
nit: Let's remove the spaces before/after the equals sign here to be consistent.